packages/common/src/validation/database.ts

```diff
@@ -26,12 +31,28 @@ const SchemaDatabaseRecordQuery = Joi.object<TDatabaseListRequest['query']>({
   }).optional(),
 });
 
-export const DatabaseListRequest = Joi.object<TDatabaseListRequest>({
+export const SchemaDatabaseList = Joi.object<TDatabaseListRequest>({
```

Comment:
When you refactor this part and move it here, I'm prefer to rename `const` to `UPPER_SNAKE_CASE` will be better, since we may have function helper for schema, so function is in `camelCase`. It will help to distingue `const` schema and `function` schema.

Reply:
Remember our naming convention, I'm not prefer `SCHEMA` as common prefix, since for `SCHEMA` builder you may have, `SCHEMA_SCHEMA` it's little bit confuse.

`JOI_` prefix is better good fit, `JOI_DATABASE_CREATE` or `JOI_SCHEMA_BUILDER` all work.